### PR TITLE
Adding a Code of Conduct to the page.

### PR DESCRIPTION
**Pull request by martinheidegger**
This is following a (internal) conversation on discord: https://discord.com/channels/1034792577293094972/1229269186849800202

Here are two screenshots.

Screenshot 2024-04-15 at 21-12-59 Osaka Kyoto Web Designers and Developers Meetup
Screenshot 2024-04-15 at 21-13-44 Osaka Kyoto Web Designers and Developers Meetup

### CODE REVIEW

File: pages/coc-report.vue [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        <ul>
+          <li>Your contact info <em>(so we can get in touch with you if we need
+            to follow up)</em></li>
+          <li>Names <em>(real, nicknames, or pseudonyms)</em> of any individuals
+            involved. If there were other witnesses besides you, please try to include
+            them as well.
+          </li>
+          <li>When and where the incident occurred. Please be as specific as possible.</li>
+          <li>Your account of what occurred. If there is a publicly available record
+            (e.g. a mailing list archive or a public IRC logger) please include
```
Comment (kinbiko) [inline pages/coc-report.vue]: This seems more relevant.
```suggestion
            (e.g. a link to a Discord message or Meetup comment) please include
```

**Review by kinbiko — APPROVED** [review]
Thank you!

File: pages/coc-report.vue (hunk 1 above)
Comment (martinheidegger) [reply pages/coc-report.vue]: Updated in https://github.com/owddm/owddm.github.io/pull/259/commits/0c4e8769e550e1145a4a509f84ff82b5d451495b

File: pages/coc-report.vue [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          your report to the committee, as all members will see the report.
+        </p>
+        <p>
+          In that case, you can make a report directly to any or all other organizers.
+        </p>
+        <p>
+          The members you contact will respond to the issue, to the best of their
+          ability, in a group excluding the member(s) that the report concerns.
+        </p>
+        -->
```
Comment (kinbiko) [inline pages/coc-report.vue]: Commented code: Is there a need to keep this in there, or can we delete it?
```suggestion
```

**Review by kinbiko — APPROVED** [review]
Looks pretty good to me!

**Comment by martinheidegger** [comment]
Review time has passed, merging it.

**Pull request merged**